<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLViewer" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>770</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>580</width>
        <height>770</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox_rotate">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>200</y>
         <width>276</width>
         <height>230</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>230</height>
        </size>
       </property>
       <property name="title">
        <string>Угол поворота °</string>
       </property>
       <widget class="QSlider" name="horizontalSlider_rotate_x">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="qlabel_rotate_x">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_rotate_y">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="qlabel_rotate_y">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_rotate_z">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="qlabel_rotate_z">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_x">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_y">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_z">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_position">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>430</y>
         <width>276</width>
         <height>230</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>230</height>
        </size>
       </property>
       <property name="title">
        <string>Местоположение</string>
       </property>
       <widget class="QSlider" name="horizontalSlider_position_z">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>-10</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_position_x">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_position_y">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="qlabel_position_z">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QLabel" name="qlabel_position_y">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_position_z">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_position_y">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="qlabel_position_x">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_position_x">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_scale">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>660</y>
         <width>276</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Масштаб</string>
       </property>
       <widget class="QSlider" name="horizontalSlider_scale">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_info">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>0</y>
         <width>276</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Информация</string>
       </property>
       <widget class="QLabel" name="label_vertex">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Количество вершин:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_vertex_count">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_edge_count">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_edge">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Количество ребер:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_file">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Название файла:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_filename">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_projection">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>276</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Проекция</string>
       </property>
       <widget class="QRadioButton" name="radioButton_central">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Центральная</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_parallel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Параллельная</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_setting_edge">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>276</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Настройки ребер:</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Толщина</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_edgeThickness">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>25.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_edgeType">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Сплошной</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Пунктирный</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Цвет</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_color">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_setting_edge_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>280</y>
         <width>276</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Настройки  вершин:</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Размер</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_vertexSize">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>25.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_vertexType">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Круг</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Квадрат</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Отсутствует</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Цвет</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_vertexColor">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_setting_edge_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>450</y>
         <width>276</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Настройки  фона:</string>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>37</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Цвет</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_backColor">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_selectFile"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_selectFile">
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewer</class>
   <extends>QOpenGLWidget</extends>
   <header>GLViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
